packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -545,11 +561,25 @@ class _RailDestination extends StatelessWidget {
     final Widget content;
     switch (labelType) {
       case NavigationRailLabelType.none:
+        final Widget iconWithIndicator = Stack(
+          alignment: Alignment.center,
+          children: <Widget>[
+            if (useIndicator)
+              NavigationIndicator(
+                animation: destinationAnimation,
```

Comment:
I think adding a param to nav rail theme for customizing the color of the indicator (like the NavigationBar does) would be useful

Reply:
Done.